js/private/BUILD.bazel

```diff
@@ -44,8 +44,8 @@ bzl_library(
     srcs = ["npm_import.bzl"],
     visibility = ["//js:__subpackages__"],
     deps = [
-        "@aspect_bazel_lib//lib:repo_utils",
```

Comment:
ahh. looks like I don't have formatting setup. I'll get the formatting right on the branch so these don't show up on this diff